homeworks/03-data-structure/src/binary-tree/binaryTree.test.ts

```diff
@@ -0,0 +1,73 @@
+import { BinaryTree, TraverseType } from './binaryTree';
+import { assertNever } from '../utils/commonUtils';
```

Comment:
'../utils/commonUtils'

If you already have `utils` folder, I would say `commonUtils` is a bit over. Just `common` is enough (or even just `utils/assetNever`) because by reading the full path it's obvious that this is `utils`.

The same works everywhere. For example, in React world `components/Header`. (no `HeaderComponent`, etc.)

Reply:
Got it, updated - https://github.com/vbarbasins/evo-ts-bootcamp/pull/3/commits/ee4d4182e8db13525b5c1da604d2d6488f473a96